overlay.yml

```diff
@@ -152,3 +152,11 @@ components:
       name: selinux-policy
       branch: master
       src: github:cgwalters/selinux-policy-centos-atomic-backport
+
+  - distgit: cpio
+
+  - src: distgit
+    distgit:
+      name: dracut
+      branch: master
+      src: github:giuseppe/dracut-cahc-distgit
```

Comment:
At some point I think what we really want here is the ability to have a patch set that's applied on top.  But this seems OK for now, the tradeoff is we'll have to remember to keep it updated.